board/safety/safety_toyota.h

```diff
@@ -249,6 +249,7 @@ static int toyota_tx_hook(CAN_FIFOMailBox_TypeDef *to_send) {
 static void toyota_init(int16_t param) {
   controls_allowed = 0;
   relay_malfunction = 0;
+  gas_interceptor_detected = 0;
```

Comment:
Is there a reason this wasn't done before?

Reply:
It's initialized in safety declarations. But good catch, this could be an issue if you unplug a pedal without restarting the panda.